CoreArrangementPlugins/src/au/gov/asd/tac/constellation/plugins/arrangements/uncollide/experimental/BoundingBox3D.java

```diff
@@ -0,0 +1,154 @@
+/*
+ * Copyright 2010-2020 Australian Signals Directorate
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package au.gov.asd.tac.constellation.plugins.arrangements.uncollide.experimental;
+
+import au.gov.asd.tac.constellation.graph.GraphElementType;
+import au.gov.asd.tac.constellation.graph.GraphReadMethods;
+import au.gov.asd.tac.constellation.graph.schema.visual.concept.VisualConcept;
+
+/**
+ * This class is designed to provide a 3D bounding box for a graph.
+ * <p>
+ * 
+ * @author algol
+ * @author Nova
+ */
+class BoundingBox3D extends AbstractBoundingBox{ 
```

Comment:
same here

Reply:
If the class were public it would need to reside somewhere different as a general graph utility class. Given the only use I can see for it is to find colliders I think it is very unlikely anything else will want to use this class, hence leaving it in its current location with the default/package private access level.